packages/airnode-deployer/terraform/airnode/aws/templates/httpGw.yaml.tpl

```diff
@@ -59,9 +59,6 @@ components:
       name: x-api-key
       in: header
 
-security:
-  - apiKey: []
-
```

Comment:
I had to remove this at this level since the browser preflight checks do not include the apiKey